src/mol-math/linear-algebra/3d/vec3.ts

```diff
@@ -474,7 +474,9 @@ namespace Vec3 {
 
     /** Computes the angle between 2 vectors, reports in radians. */
     export function angle(a: Vec3, b: Vec3) {
-        const theta = dot(a, b) / Math.sqrt(squaredMagnitude(a) * squaredMagnitude(b));
+        const denominator = Math.sqrt(squaredMagnitude(a) * squaredMagnitude(b));
+        if (denominator === 0) return Math.PI / 2;
```

Comment:
Would ``denominator <= epsilon`` produce more stable results?

Reply:
maybe, the current way is like three.js is doing it

Reply:
Ok I guess we could leave it as it is.